mattermost.go

```diff
@@ -134,6 +134,7 @@ func main() {
 
 		setDiagnosticId()
 		go runSecurityAndDiagnosticsJob()
+		go resetStatuses()
```

Comment:
It should be remedied pretty quick as soon as they click on something in the app, but to be more correct and accurate I can move it